static/js/settings_account.js

```diff
@@ -341,6 +341,18 @@ exports.set_up = function () {
         });
     });
 
+    $("#user_avatar_delete_button").hover(function () {
+        $("#user_avatar_upload_button").text(i18n.t("Delete profile picture"));
+        $("#user_avatar_upload_button").css({right: '40px'});
+        $("#user_avatar_delete_button").css({color: "hsl(0, 0%, 100%)"});
+    }, function () {
+        $("#user_avatar_upload_button").text(i18n.t("Upload new profile picture"));
+        $("#user_avatar_upload_button").css({right: '20px'});
+        $("#user_avatar_delete_button").css({color: "hsl(0, 0%, 75%)"});
+    }
+    );
```

Comment:
We generally don't do direct CSS changes in JS.  Instead, we can just do `:hover` CSS rules to detect hover.  (And the text can be two strings in the template, with CSS controlling which is visible on-hover)

Also, why the 20px/40px numbers?  Is that to do centering?  We should just have CSS that centers it, that's much more reliable.

Reply:
ok Tim, I'll change